### PR TITLE
Use original source for evaluating call objects

### PR DESCRIPTION
**Pull request by jeroen**
Alternate solution to: https://github.com/hadley/evaluate/issues/74

### CODE REVIEW

File: R/parse.r [hunk 1]
```diff
@@ -170,3 +170,13 @@ parse_all.default <- function(x, filename = NULL, ...) {
...
   parse_all(deparse(x), filename, ...)
 }
+
+# Calls are already parsed and always length one
+#' @export
+parse_all.call <- function(x, filename = NULL, ...) {
+  expr <- list(as.expression(x))
+  src <- deparse(x)
+  out <- data.frame(src = src, stringsAsFactors = FALSE)
+  out$expr = expr
```
Comment (yihui) [inline R/parse.r]: `=` to `<-`

**Comment by jeroen** [comment]
OK done, thanks :)

**Comment by jeroen** [comment]
Actually can you hold on merging, I need to test one more thing.

**Comment by jeroen** [comment]
OK this works as expected for me, thanks!

File: DESCRIPTION [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Package: evaluate
 Type: Package
 Title: Parsing and Evaluation Tools that Provide More Details than the Default
-Version: 0.11
```
Comment (jeroen) [inline DESCRIPTION]: Ah sorry I don't know your versioning preference

Comment (yihui) [inline DESCRIPTION]: No worries at all. It is a trivial matter.

**Comment by jeroen** [comment]
Would you have time to do a release any time soon? I would love to use start using this feature, and there is no easy way to have a temporary workaround because UTF8 code actually breaks when it's deparsed.

One more thing, maybe we should add `S3method(parse_all,call)` to the `NAMESPACE` ?

**Comment by yihui** [comment]
Just ran roxygen to update NAMESPACE.

I will start the revdep check soon and make a new release.

**Comment by yihui** [comment]
Uploaded to CRAN.

**Comment by jeroen** [comment]
That was fast, thanks!

**Comment by yihui** [comment]
On CRAN now :)

**Pull request merged**